library/core/src/mem/valid_align.rs

```diff
@@ -46,6 +48,13 @@ impl ValidAlign {
     pub(crate) fn log2(self) -> u32 {
         self.as_nonzero().trailing_zeros()
     }
+
+    /// Returns the alignment for a type.
+    #[inline]
+    pub(crate) fn of<T>() -> Self {
```

Comment:
I had this locally in an earlier version of this type, but deleted it when I made it `pub(crate)` 🙃